tensorflow/core/platform/default/logging.cc

```diff
@@ -117,6 +127,8 @@ LogMessage::~LogMessage() {
   if (TF_PREDICT_TRUE(severity_ >= min_log_level)) GenerateLogMessage();
 }
 
+int LogMessage::min_lvl_ = MinVLogLevel();
```

Comment:
The way this is written, this is called during binary initialization.  It might be nice to have the semantics match the existing normal log behavior, which is that the environment variable is read only the first time you call VLOG.

So what do you think about doing the following: make min_lvl_ a function in LogMessage():

int64 LogMessage::min_vlog_level() {
  static int64 min_vlog_level = MinLogLevel();
  return min_vlog_level;
}

and then call LogMessage::min_vlog_level() in VLOG_IS_ON ?